LemonWay/LemonWayAPI.php

```diff
@@ -975,4 +975,22 @@ private function cleanRequest($str)
 
         return $str;
     }
+
+    /**
+     * Allows us to call methods with first character in uppercase.
+     *
+     * @param  string $methodName
+     * @param  array $arguments
+     * @return mixed
+     */
+    public function __call($methodName, $arguments)
+    {
+        if (method_exists($this, lcfirst($methodName))) {
+            return call_user_func_array([$this, lcfirst($methodName)], $arguments);
+        }
+
+        $className = static::class;
+
+        throw new \BadMethodCallException("Call to undefined method {$className}::{$methodName}()");
+    }
```

Comment:
I don't think this is really useful, because PHP function calls are not case sensitive

> Note: Function names are case-insensitive, though it is usually good form to call functions as they appear in their declaration.

http://fr.php.net/manual/en/functions.user-defined.php